.github/workflows/node.js.yml

```diff
@@ -7,11 +7,7 @@
 
 name: CI
 
-on:
-  push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+on: push
```

Comment:
Why turn off CI for pull requests?

Other than that, PR looks good and can be merged IMO.

Reply:
@staltz  it's still gonna run on commits in PRs. i think that setting is just "only run on commits part of a PR" or something.

the improvement here is that it'll run on commits even before you've opened a PR. if you look at the first commit in this PR, it doesn't have this CI because it was using the old config

another problem with the old config is that it only ran on stuff pointing at master, so if there was a PR pointing at another PR, that PR wouldn't get any CI

Reply:
I think this only works for PRs created as branches in this repo. If an external contributor wants to send a PR, CI wouldn't run for those.

Reply:
Ok there doesn't seem to be a good solution here which really sucks https://github.com/orgs/community/discussions/26276 why is github missing such a basic feature :cry: 

Anyway reverted this now